### PR TITLE
Iconfont support

### LINKED ISSUE #78: Integrate icon webfont generation within gulp
- https://github.com/backflip/gulp-iconfont
- https://github.com/backflip/gulp-iconfont-css

### PR DESCRIPTION
**Pull request by FinalAngel**
fixes #78

### CODE REVIEW

**Comment by vxsx** [comment]
sweet

**Comment by vxsx** [comment]
:shipit:

File: gulpfile.js [hunk 1]
```diff
@@ -110,6 +114,24 @@ gulp.task('docs', function () {
...
+            fontName: 'iconfont',
+            fontPath: 'static/fonts/iconfont/',
+            path: PROJECT_PATH.sass + 'libs/_iconfont.scss',
+            targetPath: '../../../private/sass/layout/_iconfont.scss'
+        }))
+        .pipe(iconfont({
+            fontName: 'iconfont',
+            normalize: true
+        }))
+        .on('glyphs', function(glyphs, options) {
```
Comment (sergii-kulakovskyi) [inline gulpfile.js]: space after `function`

**Comment by lory87** [comment]
:+1:

**Comment by sergii-kulakovskyi** [comment]
:+1:

**Comment by evildmp** [comment]
Looks good!

**Pull request merged**